mLRS/Common/hal/rx-hal-R9M-868-f103c8.h

```diff
@@ -239,8 +239,10 @@ void rfpower_calc(int8_t power_dbm, uint8_t* sx_power, int8_t* actual_power_dbm,
     voltage_mV = 1730;
     *actual_power_dbm = 27;
   } else if (power_dbm > 22) {
-    voltage_mV = 1390;
-    *actual_power_dbm = 23;
+    //voltage_mV = 1390;
+    //*actual_power_dbm = 23;
+    voltage_mV = 1475;
```

Comment:
did you guesstimate this value, or did you had a chance to adjust it using a power meter?

Reply:
Guessed 1450, needed 1475 to be consistently over 24 dBm on the IRC power meter.  I now recall there were concerns on R9 overheating, can lower have a safer value if we want.